enhancements/installer/openstack-for-power-vc-ipi.md

```diff
@@ -0,0 +1,300 @@
+there are core differences that we'll need to address in the installer.
+PowerVC handles images in a very specific way, so we'll need to perform
+some extra steps to make the images bootable. Additionally, we'll need
+to use swift to host ignition as glance is configured in a way that
+we cannot guarantee the file can be downloaded.
+
+### Risks and Mitigations
+
+While the additional functionality will not pose a risk to other
+teams, it will expand what is tested and supported by the IBM team. We are
+confident that it will fit in our existing team so the risk is minimal.
+
+### Drawbacks
+
+There will be some prerequisites expected from the user, however this
+is not unheard of in the installer.
+ 
+### Implementation Details/Notes/Constraints
+We will need to work around some features that the existing OpenStack
+platform uses that we do not support. For example, we'll need to avoid
+using security groups. We've proven this can be done.
+
+#### Features used by the OpenStack Provider that are Not Supported in PowerVC 
+1. Security Groups
+2. Allowed Address Pairs (Initial load balancing)
+3. Native DHCP networks
+
+## Design Details
+
+Basic Walkthrough:
+1. The user creates a DHCP network in PowerVC.
+2. The user prepares the clouds.yaml with access information to their PowerVC environment. 
+3. The installer uploads an OVA from IBM Cloud COS and makes it bootable.
+4. The installer hands the install over to CAPO.
+
+
+[Power VS Reference Doc][power-vs-reference-doc]
+
+### Installation Configuration
+Here is a sample of an **install-config.yaml** for a PowerVC installation.
+
+
+```yaml
+apiVersion: v1
+baseDomain: example.com
+compute:
+- architecture: ppc64le
+  hyperthreading: Enabled
+  name: worker
+  platform:
+    powervc:
+      zones:
+        - e980
+  replicas: 3
+controlPlane:
+  architecture: ppc64le
+  hyperthreading: Enabled
+  name: master
+  platform:
+    powervc:
+      zones:
+        - e980
+  replicas: 3
+metadata:
+  creationTimestamp: null
+  name: powervc-cluster 
+networking:
+  clusterNetwork:
+  - cidr: 10.128.0.0/14
+    hostPrefix: 23
+  machineNetwork:
+  - cidr: 10.20.176.0/20
+  networkType: OVNKubernetes
+  serviceNetwork:
+  - 172.30.0.0/16
+platform:
+  powervc:
+    loadBalancer:
+      type: UserManaged
+    apiVIPs:
+    - 10.20.184.56
+    cloud: powervc
+    defaultMachinePlatform:
+      type: project-name
+    ingressVIPs:
+    - 10.20.184.56
+    machinesSubnet: 8f895eaa-d54f-4c2e-8f00-e1c5eeb51aa1
+publish: External
+pullSecret: ''
+sshKey: ''
+```
+
+### Networking
+Because PowerVC does not have native DHCP network support, we are requiring admins set up their own DHCP network service before installing.
+
+#### Load Balancing
+The OpenStack provider uses the "allowed-address-pairs" feature of Neutron for initial loadbalancing. Because PowerVC lacks this feature, we expect admins to onboard their own load balancers as a day-2 operation.
```

Comment:
Many questions about this.

A load balancer for the API is required during installation, so... how is that going to work if users don't set up a load balancer until day 2?

Reply:
@zaneb I was wrong about this, I had thought @hamzy's prototype came up without an LB but I was wrong.

I will clarify in the next revision. Yes, we'll need something to loadbalance